views/common/AllStations.js

```diff
@@ -0,0 +1,39 @@
+import { Pressable, Text, StyleSheet } from "react-native";
+import { useState } from "react";
+import StopModal from "./StopModal";
+
+export default function AllStations(props) {
+  return(props.displayStations.map(function callback(item) {
+    const inFavorites = props.currentFavorites.includes(item["GTFS Stop ID"]);
+  return(<Station key={item["GTFS Stop ID"]} daytimeRoutes={item["Daytime Routes"]} stopName={item["Stop Name"]} stopId={item["GTFS Stop ID"]} inFavorites={inFavorites} changeLastUpdated={props.changeLastUpdated}/>)
```

Comment:
Can we fix indent here? Can we also break this into multiple lines (it's a little bit very long)?